.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Master
+
+on:
+  push:
+    branches:
+      - master
+
+  # This allows us to manually run this job
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    if: github.repository == 'IABTechLab/uid2-android-sdk' && github.ref == 'refs/heads/master'
```

Comment:
This line does the following:
 - Makes sure that anyone who forks the repo doesn't attempt to publish (even though they can't, because they don't have our credentials)
 - An additional check to confirm that we only publish from `master`.

The second is arguably overkill, but I saw other examples in popular repos that do this, so decided to keep to be consistent.

Reply:
Branch is `main`.